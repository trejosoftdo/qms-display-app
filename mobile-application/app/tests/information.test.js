import React from 'react';
import Information from '../information';


describe('Information route component', () => {
  it('renders the component as expected', () => {
    expect(renderShallow(<Information  />)).toMatchSnapshot();
  });
});
