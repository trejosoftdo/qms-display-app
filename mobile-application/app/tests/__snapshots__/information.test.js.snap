// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Information route component renders the component as expected 1`] = `
<React.Fragment>
  <AppHeader
    title="Translated[translation:information]({})"
  />
  <InformationView />
</React.Fragment>
`;
